wp-content/themes/reconasia/assets/_js/custom/navbar.js

```diff
@@ -6,12 +6,34 @@
 const trigger = document.querySelector('.site-nav__trigger')
 const menu = document.querySelector('.site-nav__container')
 const nav = document.querySelector('.site-nav__menu')
+const parentMenus = document.querySelectorAll('.sub-menu')
 
 const Navigation = () => {
   toggleMenu()
   closeMenu()
 }
 
+parentMenus.forEach((parentMenu) => {
```

Comment:
I would recommend putting this into a function that is called by `Navigation` to keep the functionality in this file consistent.